.travis/before_script_test.sh

```diff
@@ -1,3 +1,9 @@
 #!/usr/bin/env bash
 set -ev
+TESTING_SCRIPTS_DIR=vendor/symfony-cmf/testing/bin
```

Comment:
IMO this file would be overridden by dev kit afterwards...

And can you switch back the chmod of this file please?

Reply:
No, i can not. Have the files non executeable, which is 644, breaks the check and validation for their existence. So a `+x` solved it:

```bash
maximilian@maximilian-ThinkPad-T470p:~/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle$ sudo chmod +x .travis/*
maximilian@maximilian-ThinkPad-T470p:~/OpenSource/SonataProject/SonataDoctrinePhpcrAdminBundle$ if [ -x .travis/before_install_test.sh ]; then echo "GEHT"; fi;
GEHT
```